messages/lightning.preview.site.md

```diff
@@ -1,38 +1,35 @@
 # summary
 
-Preview Experience Cloud Sites.
+Preview an Experience Cloud site locally and in real-time, without deploying it.
 
 # description
 
-In dev preview mode, you can edit local files and see these changes to your Lightning Web Components (LWC) within your site:
+Enable Lightning Preview to see local changes to your site in a real-time preview that you don't have to deploy or manually refresh. To let you quickly iterate on your Lightning web components (LWCs) and pages, your site preview automatically refreshes when Lightning Preview detects source code changes.
+
+When you edit local files with Lightning Preview enabled, your LWCs reflect the following changes in your org. To apply any other local changes not listed here, you have to deploy them to your org using the `project deploy start` command.
```

Comment:
I'm not 100% on the phrasing of this line. What are we trying to convey? Seems like the list of limitations should come directly after the first sentence. Also we should talk about the changes being reflected locally in your site, not your org specifically. 

We should also probably mention that for experience sites, you need to be publish the site first with Lightning Preview enabled before you can use lightning preview site.

Reply:
I think it'd help to outline what gets deployed automatically vs manually, and also note when you have to retrieve changes. Maybe we can reorganize this info if it's kind of unclear right now. Does this look better to you?

When you edit local files with Lightning Preview enabled, your LWCs reflect these changes in your org. 

- Basic HTML and CSS edits
- Importing new CSS-only LWCs
- JS edits in-service component library
- JS method changes that don't affect the LWC's public API

To apply any other local changes not listed here, you have to manually deploy them to your org using the `project deploy start` command.

When you make changes directly in your org, (like saving new component properties,) they're automatically deployed to your live, published site. To update your project on your local machine with those changes, you have to retrieve them from your org using the `project retrieve start` command.